src/components/Helmet.re

```diff
@@ -0,0 +1,30 @@
+[@bs.deriving abstract]
+type meta = {
+  [@bs.optional]
+  name: string,
+  [@bs.optional]
+  property: string,
+  [@bs.optional]
+  content: string,
+};
+
+[@bs.module "react-helmet"] [@react.component]
+external make:
+  (
+    ~async: option(bool)=?,
+    ~base: option('a)=?,
+    ~bodyAttributes: option('a)=?,
+    ~defaultTitle: option(string)=?,
+    ~defer: option(bool)=?,
+    ~encodeSpecialCharacters: option(bool)=?,
+    ~htmlAttributes: option('a)=?,
+    ~onChangeClientState: option('a)=?,
+    ~link: option(string)=?,
+    ~style: option(string)=?,
+    ~titleTemplate: option(string)=?,
+    ~meta: array(meta)=?,
+    ~title: option(string)=?,
+    ~children: option(React.element)=?
```

Comment:
Why would you want a prop to be `option()=?` I would keep the =? since it's a shorthand for labelled arguments being optional: 

https://reasonml.github.io/docs/en/function#defining-non-mandatory-arguments